### DISCUSSION TITLE
PyInstaller compatibility Mac M1 vs M2

### DISCUSSION THREAD

**Discussion by bjorngback** [pyinstaller/pyinstaller#7508] [PyInstaller]
From what I understand executables built with PyInstaller on one version of Mac OS will not run on a newer version of Mac OS. But what about the CPU version M1 vs M2 - if I have the same Mac OS version on both machines, will an executable built on M1 work on the M2 machine?

Currently I only have an M1 machine so I'm not able to verify this with tests. I guess that it should work if the Mac OS versions are the same but I would like to confirm to make sure.

**Comment by rokm** [comment]
Applications built on M1 should run on M2 as well, and vice versa, as they are both `arm64` architecture.

**Reply by bjorngback** [reply]
Great, thanks!
